std_msgs/Header header
string intersection_id
string phase
carla_multi_vehicle_control/TrafficApproach[] approaches


